roles/openshift_logging/tasks/install_elasticsearch.yaml

```diff
@@ -35,14 +41,15 @@
     deploy_name: "{{item.1}}"
     es_node_selector: "{{openshift_logging_es_nodeselector | default({})}}"
   with_indexed_items:
-    - "{{es_dc_pool | default([])}}"
+    - "{{ es_dc_pool }}"
   check_mode: no
-  when:
-    - "{{ openshift_logging_facts.elasticsearch.deploymentconfigs.keys() | length < openshift_logging_es_cluster_size }}"
   changed_when: no
 
 # --------- Tasks for Operation clusters ---------
```

Comment:
More of an aside, we should be able to refactor this into a single task and run it twice, once for non-ops, and once for ops.